doc/fileformat/cable_cell.rst

```diff
@@ -0,0 +1,429 @@
+   membrane capacitance,               --,             ✓,             ✓
+   ion initial internal concentration, --,             ✓,             ✓
+   ion initial external concentration, --,             ✓,             ✓
+   ion initial reversal potential,     --,             ✓,             ✓
+   ion reversal potential method,      --,            --,             ✓
+   density mechanism,                  --,             ✓,            --
+   point mechanism,                    ✓,             --,            --
+   current clamp,                      ✓,             --,            --
+   threshold detector,                 ✓,             --,            --
+   gap junction site,                  ✓,             --,            --
+
+The various properties and dynamics of the decor are described as follows:
+
+.. label:: (membrane-potential val:real)
+
+   This describes an *initial membrane potential* object with value ``val`` (unit [mV]).
+
+.. label:: (axial-resistivity val:real)
+
+   This describes an *axial resistivity* object with value ``val`` (unit [Ω·cm]).
+
+.. label:: (temperature-kelvin val:real)
+
+   This describes a *temperature* object with value ``val`` (unit [K]).
+
+.. label:: (membrane-capacitance val:real)
+
+   This describes a *membrane capacitance* object with value ``val`` (unit [F/m²]).
+
+.. label:: (ion-internal-concentration ion:string val:real)
+
+   This describes an *initial internal concentration* object for ion ``ion`` with value ``val`` (unit [mM]]).
+
+.. label:: (ion-external-concentration ion:string val:real)
+
+   This describes an *initial external concentration* object for ion ``ion`` with value ``val`` (unit [mM]]).
+
+.. label:: (ion-reversal-potential ion:string val:real)
+
+   This describes an *initial reversal potential* object for ion ``ion`` with value ``val`` (unit [mV]]).
+
+.. label:: (mechanism name:string [...(param:string val:real)])
+
+   This describes a (point or density) mechanism object of the mechanism called ``name``. This expression
+   accepts zero or more ``(param:string val:real)`` expressions. Each of these expressions sets the value of
+   parameter ``param`` to ``val``.
+   For example:
+
+   .. code:: lisp
+
+      (mechanism "hh" ("gl" 0.5) ("el" 2))
+
+   This expression creates an "hh" mechanism and sets the "gl" and "el" parameters of the mechanism to 0.5
+   and 2 respectively (units depend on the :ref:`nmodl <formatnmodl>` mechanism).
+
+.. label:: (ion-reversal-potential-method ion:string method:mechanism)
+
+   This creates a *reversal potential method* (able to modify the reversal potential) of ion ``ion`` from
+   mechanism ``method``.
+   For example:
+
+   .. code:: lisp
+
+      (ion-reversal-potential-method "ca" (mechanism "nersnt/ca"))
+
+.. label:: (current-clamp (envelope-pulse delay:real duration:real amplitude:real) freq:real)
+
+   This creates a *current clamp*. If the frequency ``freq`` (unit [Hz]) is zero, the current is a square
```

Comment:
Probably not related to this PR as such, and I hadn't looked at the clamps yet, but elsewhere I believe the unit for frequencies are [kHz], which in turn matches [ms] better than [s]. Would it be possible to use [kHz] here?

Reply:
You're right, we should be using kHz. Sam is going to fix this and the underlying class units in a future PR.

Reply:
OK! Will that follow this PR quickly? If not, we'll need to bump the version quite soon for a reason we already foresee, which is a tiny bit unfortunate :) If we're sure of that other PR, then can we do it 'dirty' and divide/multiply outputs/inputs right now, in order to keep the format unchanged?

Reply:
It should follow quite soon. We'll have to make another change to the format (adding a `phase` parameter to the current-clamp) anyway once that future PR is in. I don't think this will be a huge issue, I don't know if we'll need to bump the version for example.

Reply:
:+1: